packages/fxa-content-server/app/scripts/views/sms_send.js

```diff
@@ -31,11 +31,28 @@ class SmsSendView extends FormView {
   mustAuth = true;
   template = Template;
 
+  afterRender() {
+    // Make this a jQuery object so we can use the validation helpers
+    this.phoneField = this.$el.find('.phone-number');
```

Comment:
Turns out we already have a [jQuery-based validation method](https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/app/scripts/views/elements/tel-input.js#L25-L38) that can be utilized. I am happy to break off of this if desired.

Reply:
Awesome find! I tested a bunch of different stuff in here and it seems to work very well. Alex will be happy. :) 

We probably want to run `_validatePhoneNumber` when the user redirects back to `sms` as well, otherwise when you click "Wrong number?" from `sms/sent`, the button is disabled despite being prefilled with the number just sent. I think we probably also want a disabled grey style based on the feature doc.

Also, will you squash your first 4 commits?

Reply:
Done and done.